docs/src/components/Layout/Footer.tsx

```diff
@@ -1,27 +1,71 @@
-          policy.
-        </Text>
-        <Link
+    <Flex as="footer" direction="column" className="docs-footer">
+      <Flex direction="row" justifyContent="center">
+        <Logo />
+      </Flex>
+      <View textAlign="center">
+        <Button
+          as="a"
+          variation="link"
+          gap={tokens.space.small}
+          alignItems="baseline"
           href="https://github.com/aws-amplify/amplify-ui"
           isExternal={true}
+          color={tokens.colors.font.tertiary}
+        >
+          <Icon ariaLabel="Github" pathData={icons.GITHUB} />
+          Github
+        </Button>
+        <Button
+          as="a"
+          variation="link"
+          gap={tokens.space.small}
+          alignItems="baseline"
+          isExternal={true}
+          color={tokens.colors.font.tertiary}
         >
           <Icon
-            ariaLabel="Github"
-            color="var(--amplify-colors-font-tertiary)"
-            pathData="M15.4992327,3 C8.59699202,3 3,8.5085046 3,15.3048195 C3,20.7400586 6.58118478,25.3512503 11.5489564,26.9789518 C12.1743401,27.0914936 12.4022406,26.7115707 12.4022406,26.385275 C12.4022406,26.0937239 12.3914979,25.3195271 12.3853591,24.2930554 C8.90853284,25.0362843 8.17495396,22.6434498 8.17495396,22.6434498 C7.60635359,21.2219489 6.78683241,20.8442919 6.78683241,20.8442919 C5.6519337,20.0806696 6.87277471,20.0957758 6.87277471,20.0957758 C8.12737876,20.1833922 8.78729282,21.3639479 8.78729282,21.3639479 C9.90224064,23.2439244 11.7124002,22.7008536 12.4252609,22.386643 C12.5388275,21.5912974 12.8611111,21.048982 13.2186924,20.7415693 C10.4432167,20.4311352 7.52501535,19.3752063 7.52501535,14.6605367 C7.52501535,13.3175885 8.01227747,12.219362 8.81184776,11.3590594 C8.68293432,11.0478701 8.25399018,9.79707019 8.93385513,8.10290101 C8.93385513,8.10290101 9.98357888,7.7720735 12.371547,9.3642753 C13.3683241,9.09085165 14.4379988,8.95489514 15.5007673,8.94960794 C16.5620012,8.95489514 17.6316759,9.09085165 18.6299877,9.3642753 C21.0164211,7.7720735 22.0638428,8.10290101 22.0638428,8.10290101 C22.7460098,9.79707019 22.3170657,11.0478701 22.1881522,11.3590594 C22.9892572,12.219362 23.47345,13.3175885 23.47345,14.6605367 C23.47345,19.3872913 20.5506446,20.4273587 17.7659607,20.7317502 C18.2148557,21.1116731 18.6146409,21.8624551 18.6146409,23.0105323 C18.6146409,24.6548508 18.599294,25.9819374 18.599294,26.385275 C18.599294,26.7145919 18.8241252,27.0975361 19.458717,26.9774412 C24.4211172,25.3467184 28,20.738548 28,15.3048195 C28,8.5085046 22.403008,3 15.4992327,3"
+            viewBox={{ minX: 0, minY: 0, width: 245, height: 240 }}
+            pathData={icons.DISCORD}
           />
-        </Link>
+          Discord
+        </Button>
+      </View>
+      <Flex direction="row">
+        <View>
+          Amplify open source, documentation and community are supported by
+          Amazon Web Services © {new Date().getFullYear()}, Amazon Web Services,
```

Comment:
Thank you for not making us update the copyright year every December 🤣

Reply:
lol